cocos2d/core/labelttf/CCLabelTTFCanvasRenderCmd.js

```diff
@@ -460,15 +472,14 @@ cc.LabelTTF._firsrEnglish = /^[a-zA-Z0-9ÄÖÜäöüßéèçàùêâîôû]/;
     proto._updateTexture = function () {
         this._dirtyFlag = this._dirtyFlag & cc.Node._dirtyFlags.textDirty ^ this._dirtyFlag;
         var node = this._node;
-        var locContentSize = node._contentSize;
+        var scale = cc.view.getDevicePixelRatio();
         this._updateTTF();
-        var width = locContentSize.width, height = locContentSize.height;
         if (node._string.length === 0) {
-            node.setTextureRect(cc.rect(0, 0, 1, locContentSize.height));
+            node.setTextureRect(this._texRect);
             return true;
         }
         this._saveStatus();
-        node.setTextureRect(cc.rect(0, 0, width, height));
+        node.setTextureRect(this._texRect);
```

Comment:
But it's being overwritten here. See the default implementation of setTextureRect() in CCSprite: https://github.com/cocos2d/cocos2d-html5/blob/develop/cocos2d/core/sprites/CCSprite.js#L738